www/components/CardBoard/CardBoard.module.css

```diff
@@ -7,4 +7,5 @@
 
 .card_container {
   margin: .5em 0 .5em 0;
+  
```

Comment:
Before you push your commits you should be asking yourself if changes like these are really necessary to push along with it. I am **VERY** dissapointed

Reply:
I'm sorry this won't happen again 🖖🏻.